packages/router/README.md

```diff
@@ -57,6 +57,72 @@ they will be redirected to the route passed as the `unauthenticated` prop and th
 Redwood uses the `useAuth` hook under the hood to determine if the user is authenticated.
 Read more about authentication in redwood [here](https://redwoodjs.com/tutorial/authentication).
 
+## Sets of Routes
+
+You can group routes into sets using the `Set` component. `Set` allows you to wrap a set of Routes in another component or component(s)—usually a Context, a Layout, or both:
```

Comment:
Just "component(s)" or "component or components". With the current wording the (s) isn't really optional

Reply:
Nice totally missed that